patches/chrome-common-chrome_constants.cc.patch

```diff
@@ -1,18 +1,18 @@
 diff --git a/chrome/common/chrome_constants.cc b/chrome/common/chrome_constants.cc
-index d4e0823a55994118bacfe4fd7f8ec1881640e8ac..ce405ac073c10aea07aa2c7a5d24f8385da8d36f 100644
+index d4e0823a55994118bacfe4fd7f8ec1881640e8ac..a106021688d8b5355a07e60ad7191dd8bd216df0 100644
 --- a/chrome/common/chrome_constants.cc
 +++ b/chrome/common/chrome_constants.cc
 @@ -10,11 +10,11 @@
  #define FPL FILE_PATH_LITERAL
  
  #if defined(OS_MACOSX)
 -#define CHROMIUM_PRODUCT_STRING "Chromium"
-+#define CHROMIUM_PRODUCT_STRING "Brave"
++#define CHROMIUM_PRODUCT_STRING BRAVE_PRODUCT_STRING
```

Comment:
Framework/Helper app's name reflects `PRODUCT_FULL_NAME` in `BRANDING` file.
(Ex, `Brave-Browser-Dev Framework.framework`, `Brave-Browser-Dev Helper.app`).
Their framework/helper name is made by using `CHROMIUM_PRODCUT_STRING` and `PRODUCT_STRING`.

Reply:
can't we just use a chromium_src override here to redefine?

Reply:
I'm pretty sure this entire patch file can be replaced with overrides cc @bbondy

Reply:
done.